modules/60crypt-ssh/dropbear-stop.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh
 
+[ -f /tmp/enabled_tiocsti ] && {
+  echo 0 > /proc/sys/dev/tty/legacy_tiocsti
+}
+
```

Comment:
(Coupled with change to `dropbear-start.sh`)
```suggestion
if [ -r /tmp/legacy_tiocsti.default ] && [ -w /proc/sys/dev/tty/legacy_tiocsti ]; then
  cat /tmp/legacy_tiocsti.default > /proc/sys/dev/tty/legacy_tiocsti
fi
```